src/Chirp.Razor/DbInitializer.cs

```diff
@@ -696,9 +697,26 @@ public static void SeedDatabase(ChirpDBContext chirpContext)
             a11.Cheeps = new List<Cheep>() { c656 };
             a12.Cheeps = new List<Cheep>() { c657 };
 
+            // Set id for authors to allow insert
+            chirpContext.Database.OpenConnection();
+
+            chirpContext.Database.ExecuteSqlRaw("SET IDENTITY_INSERT Authors ON");
             chirpContext.Authors.AddRange(authors);
+            chirpContext.SaveChanges();
+            chirpContext.Database.ExecuteSqlRaw("SET IDENTITY_INSERT Authors OFF");
+
+
+            chirpContext.Database.ExecuteSqlRaw("SET IDENTITY_INSERT Cheeps ON");
             chirpContext.Cheeps.AddRange(cheeps);
             chirpContext.SaveChanges();
+            chirpContext.Database.ExecuteSqlRaw("SET IDENTITY_INSERT Cheeps OFF");
+
+            chirpContext.Database.CloseConnection();
+            
+
+            // chirpContext.Authors.AddRange(authors);
+            // chirpContext.Cheeps.AddRange(cheeps);
+            // chirpContext.SaveChanges();
```

Comment:
Can you please explain what theese "SET IDENTITY_INSERT Cheeps ON / OFF" does? :)